tests/verify-role-failed.yml

```diff
@@ -14,7 +14,7 @@
           __snapshot_failed_params.get('snapshot_lvm_percent_space_required')
           }}"
         snapshot_lvm_all_vgs: "{{
-          __snapshot_failed_params.get('snapshot_all')
+          __snapshot_failed_params.get('snapshot_lvm_all_vgs') | d(false)
```

Comment:
@richm how should I default this to false when snapshot_lvm_all_vgs is not defined.  Currently it defaults to "" and that causes an error when the module expects it to be a boolean.

Reply:
```suggestion
          __snapshot_failed_params.get('snapshot_lvm_all_vgs', false)
```